src/plotting.py

```diff
@@ -14,15 +14,19 @@ def plot_marginals(
     true_values: pd.Series = None,
 ):
     """Plot marginal ability parameters against true values."""
-    qs = infd.posterior[var_name].to_series().unstack().quantile([0.1, 0.9]).T
-    qs["truth"] = true_values
-    qs["truth_in_interval"] = (qs["truth"] < qs[0.9]) & (qs["truth"] > qs[0.1])
-    qs = qs.sort_values(0.9)
-    y = np.linspace(*ax.get_ylim(), len(qs))
+    qs = infd.posterior[[var_name]].quantile([0.1, 0.9], dim=("chain", "draw"))
+    qs["truth"] = (("skater_name",), true_values)
+    qs["truth_in_interval"] = (
+        (qs["truth"] < qs[var_name].sel(quantile=0.9)) &
+        (qs["truth"] > qs[var_name].sel(quantile=0.1))
+    )
+    qs = qs.sortby(qs[var_name].sel(quantile=0.9))
+    y = np.linspace(*ax.get_ylim(), qs.dims["skater_name"])
```

Comment:
This is probably the piece that is more xarray specific and can be confusing. `qs.dims["skater_name"]` returns an the length of the dimension as integer. The key is in the difference between dimension and coordinate. Here both things have the same name, like in all ArviZ generated datasets but this is not a requirement in xarray. `qs.skater_name` or `qs.coords.skater_name` will return a  DataArray with all the skater name values.